x-pack/plugin/otel-data/src/main/resources/component-templates/otel@mappings.yaml

```
@@ -0,0 +1,63 @@
+---
+version: ${xpack.oteldata.template.version}
+_meta:
+  description: Default mappings for all OpenTelemetry data streams
+  managed: true
+template:
+  mappings:
+    date_detection: false
+    dynamic: false
```

Let's enable synthetic _source for all otel signals from the beginning. Based on the recent enhancements to synthetic _source, there shouldn't be any remaining blockers.

```suggestion
    dynamic: false
    _source:
      mode: synthetic
```